packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/utils/getAutoNumberingListStyle.ts

```diff
@@ -116,7 +116,7 @@ export default function getAutoNumberingListStyle(
     //The index is always the character before the last character
     const listIndex = trigger[trigger.length - 2];
 
-    if (previousListChain) {
+    if (previousListChain && parseInt(listIndex) > 1) {
```

Comment:
parseInt(listIndex) is also used in line 124, so save the result to a local var.